openapi-validator/src/test/java/io/ballerina/openapi/validator/tests/PreValidationTests.java

```diff
@@ -96,6 +96,6 @@ public void validatorTurnOff() {
         DiagnosticResult diagnostic = getCompilation(project);
         Object[] errors = diagnostic.diagnostics().stream().filter(d ->
                         DiagnosticSeverity.WARNING == d.diagnosticInfo().severity()).toArray();
-        Assert.assertEquals(errors.length, 5);
+        Assert.assertEquals(errors.length, 3);
```

Comment:
What are we asserting in here? Don't we need to assert the specific error messages?

Reply:
This test is regarding the openAPI validator. currently, this test set hasn't had a message comparison process. Will improve all the test sets with separate PR once applying that process.

Reply:
asserting only the diagnostics count might not be that much accurate IMO. Better to revisit the test suite and fix the relavant places like you suggested

Reply:
Yeah, + 1 for the suggestion as discussed in the past meetings, we came up with a decision to find message comparison according to the feature behaviour so far only remaining to improve Validator tests, other feature tests have already improved. since this is the Validator test shall we address this when revisiting the test related to the test improvement in Validator?